### ISSUE TITLE
Masked face

### ISSUE THREAD

**Issue by lavrenkov-sketch** [swghosh/DeepFace#12]
Hello really build version with face recognition with mask?

**Comment by swghosh** [comment]
This project was developed at a time well before we hit the pandemic. Nevertheless, I am skeptical that the model (as it is trained at million scale) would be robust to such occlusions like face masks. But, it might miss out on a lot as well!

Probably, give it a shot and why not try to finetune the pre-trained model on a face dataset with many pictures that includes masks?
